index.css

```diff
@@ -0,0 +1,37 @@
+#main{
+    background-color: #EEEEEE;
+    padding: 15px;
+}
+
+body{
+    text-align: center ;
+}
+
+#title{
+    text-align: center;
+}
+
+#img-div{   
+    padding: 10px;
+    background-color: white;
+}
+#image{
+    max-width: 100%;
+    height: auto;
+}
```

Comment:
This is top tier styling Francis but I think it would better if you remember to add a space after identifier name to prevent stylelint errors.